docs/native-modules-autolinking.md

```diff
@@ -0,0 +1,64 @@
+---
+id: native-modules-autolinking
+title: Autolinking Native Modules
+---
+
+Autolinking is a mechanism that allows your React Native app project to discover and use native modules and view managers provided by React Native libraries.
+
+This document covers autolinking for the Windows platform. It is an extension to the [React Native CLI Autolinking doc](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md).
+
+Add a library using your favorite package manager and run the build:
+
+```
+yarn add react-native-webview
+npx react-native run-windows
+```
+
+That's it. No more editing native files to use native code.
+
+## How does it work
+
+From the [React Native CLI Autolinking doc](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md#how-does-it-work):
+
+> Each platform defines its own [`platforms`](https://github.com/react-native-community/cli/blob/master/docs/platforms.md) configuration. It instructs the CLI on how to find information about native dependencies. This information is exposed through the [`config`](https://github.com/react-native-community/cli/blob/master/docs/commands.md#config) command in a JSON format. It's then used by the scripts run by the platform's build tools. Each script applies the logic to link native dependencies specific to its platform.
+
+The information provided by `config` is described in [React Native Config Schema](config.md).
+
+## Autolinking process
+
+Autolinking is performed automatically as a part of the `run-windows` command:
+
+1. At build time, autolinking is performed first, before `msbuild.exe` is invoked and the build actually started. It uses the information provided by `config` to both generate and modify certain native files consumed by your app project.
+    1. The `AutolinkedNativeModules.g.targets` file contains the necessary references to the dependency projects that must be built.
```

Comment:
Where do these files get generated, relative to the solution source directory?
Ideally they go into the Generated Files folder